docs/machine-learning/how-to-guides/load-data-ml-net.md

```diff
@@ -0,0 +1,146 @@
+---
+title: "How-To: Load Data in ML.NET"
+description: Load data file and streaming data into ML.NET
+ms.date: 04/03/2019
+ms.custom: mvc,how-to
+#Customer intent: As a developer I want to know how to load data from file and streaming data sources.
+---
+
+# How-To: Learn how to load data from file and real-time sources in ML.NET
+
+This how-to shows you how to load data for processing and training into ML.NET. The data is originally stored in files or real-time / streaming data sources.
+
+## Create the data model
+
+ML.NET enables you to define data models via classes. For example, given the following input data:
+
+```text
+Size (Sq. ft.), HistoricalPrice1 ($), HistoricalPrice2 ($), HistoricalPrice3 ($), Current Price ($)
+700, 100000, 3000000, 250000, 500000
+1000, 600000, 400000, 650000, 700000
+```
+
+Create a data model that represents the snippet below:
+
+```csharp
+public class HousingData
+{
+    [LoadColumn(0)]
+    public float Size { get; set; }
+ 
+    [LoadColumn(1, 3)]
+    [VectorType(3)]
+    public float[] HistoricalPrices { get; set; }
+
+    [LoadColumn(4)]
+    [ColumnName("Label")]
+    public float CurrentPrice { get; set; }
+}
+```
+
+### Annotating the data model with column attributes
+
+Attributes give ML.NET more information about the data model and the data source.
+
+The [`LoadColumn`](xref:Microsoft.ML.Data.LoadColumnAttribute) attribute specifies your properties' column indices.
+
+> [!IMPORTANT]
+> [`LoadColumn`](xref:Microsoft.ML.Data.LoadColumnAttribute) is only required when loading data from a file.
+
+Load columns as: 
+- Individual columns like `Size` and `CurrentPrices` in the previous `HousingData` class example.
+- Multiple columns at a time in the form of a vector like `HistoricalPrices` in the previous `HousingData` class example.
```

Comment:
>in the previous `HousingData` class examp [](start = 77, length = 41)

by previous, do you mean another article, or just the class above?

Reply:
I meant the class above